source/features/clean-conversation-filters.tsx

```diff
@@ -54,8 +54,10 @@ async function hasProjects(): Promise<boolean> {
 }
 
 async function hideProjects(): Promise<void> {
+	// TODO: False negatives require a `?.` #4884
```

Comment:
I give up 🥲 

This was the last attempt at fixing this, but: https://github.com/refined-github/refined-github/pull/4884#discussion_r726325235